examples/jsm/nodes/accessors/MaterialNode.js

```diff
@@ -113,6 +113,20 @@ class MaterialNode extends Node {
 
 			}
 
+		} else if ( scope === MaterialNode.SPECULAR_COLOR2 ) {
```

Comment:
`SPECULAR_COLOR2`? 🤔

Reply:
Exactly, `SPECULAR_COLOR` to `SPECULAR`, and `SPECULAR_COLOR2` to `SPECULAR_COLOR`. I must have forgotten this when I was finalizing the transmission PR. `SPECULAR_COLOR2` should not be the final name.

Reply:
I have changed it @sunag